scripts/travis/configure_dev.sh

```diff
@@ -54,7 +54,10 @@ if [ "${OS}" = "linux" ]; then
         fi
         set -e
         sudo apt-get update -y
-        sudo apt-get -y install sqlite3
+        sudo apt-get upgrade -y
+        sudo apt-get -y install sqlite3 libssl-dev git apt-transport-https openssl
+        git config --global http.postBuffer 524288000
+        git config --global http.sslVerify false
```

Comment:
Why do we want to disable ssl verification for git on travis?

Reply:
( it's a test. it's never intended to be submitted as such )